radicle-node/src/worker/fetch.rs

```diff
@@ -3,17 +3,17 @@ pub use refspecs::{AsRefspecs, Refspec, SpecialRefs};
 
 pub mod error;
 
-use std::collections::{BTreeMap, HashSet};
+use std::collections::{BTreeMap, BTreeSet, HashSet};
 use std::ops::Deref;
 
 use radicle::crypto::{PublicKey, Unverified, Verified};
-use radicle::git::url;
+use radicle::git::{url, Namespaced};
 use radicle::prelude::{Doc, Id, NodeId};
 use radicle::storage::git::Repository;
 use radicle::storage::refs::IDENTITY_BRANCH;
 use radicle::storage::{Namespaces, RefUpdate, Remote, RemoteId};
 use radicle::storage::{ReadRepository, ReadStorage, WriteRepository, WriteStorage};
-use radicle::{git, Storage};
+use radicle::{debug_assert_matches, git, Storage};
```

Comment:
```suggestion
use radicle::{debug_assert_matches, git, Storage};
```

This can not be use without the `test` feature

It was reported by @slack-coder  on zulip